server/embed/config.go

```diff
@@ -765,6 +765,15 @@ func (cfg *Config) Validate() error {
 		return fmt.Errorf("--experimental-compact-hash-check-time must be >0 (set to %v)", cfg.ExperimentalCompactHashCheckTime)
 	}
 
+	// If `--name` isn't configured, then multiple members may have the same "default" name.
+	// When adding a new member with the "default" name as well, etcd may regards its peerURL
+	// as one additional peerURL of the existing member which has the same "default" name,
+	// because each member can have multiple client or peer URLs.
+	// Please refer to https://github.com/etcd-io/etcd/issues/13757
+	if cfg.Name == DefaultName {
+		cfg.logger.Warn("the server is using default name, which might cause error that failed the startup.")
```

Comment:
I was planning to add a comment for this, but somehow forgot to do it:(

Please change it to something like below,
```
return fmt.Errorf("it isn't recommended to use %q name, please set a value for --name. Note that etcd might run into issue when multiple members have the same default name", DefaultName)
```

Reply:
Please ignore my previous comment. We can't return an error, because etcd may fail to get started. Let's just update the message,
```
cfg.logger.Warn("it isn't recommended to use %q name, please set a value for --name. Note that etcd might run into issue when multiple members have the same default name", DefaultName)
```

Reply:
OK, I will update it later today or tomorrow

Reply:
OK, I will update it later today or tomorrow

Reply:
updated.   https://github.com/etcd-io/etcd/pull/14642
please take a look at your convenience. thanks.